ff-parser/src/main/java/org/uniprot/core/flatfile/parser/impl/cc/cclineobject/EvidencedString.java

```diff
@@ -0,0 +1,57 @@
+package org.uniprot.core.flatfile.parser.impl.cc.cclineobject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EvidencedString {
+    private final String value;
+    private final List<String> evidences = new ArrayList<>();
+
+    public String getValue() {
+        return value;
+    }
+
+    public List<String> getEvidences() {
+        return evidences;
+    }
+
+    public EvidencedString(String value, List<String> evidences) {
+        this.value = value;
+        if (evidences != null) {
+            this.evidences.addAll(evidences);
+        }
+    }
+
+    public static EvidencedString get(String text1) {
+        return new EvidencedString(text1, null);
+    }
+
+    public static EvidencedString get(String text1, List<String> strings) {
+        if (strings == null || strings.isEmpty()) {
+            return get(text1);
+        } else {
+            return new EvidencedString(text1, strings);
+        }
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Equals and Hash for the others as well?

Reply:
Those methods were already there. I just put it in a top level class.